assets/xml/overlays/ovl_Oceff_Wipe4.xml

```diff
@@ -0,0 +1,11 @@
+<Root>
+    <File Name="ovl_Oceff_Wipe4" BaseAddress="0x80BACC10" RangeStart="0x460" RangeEnd="0xF48" Segment="128">
+        <Texture Name="sScarecrowsSongTex" OutName="scarecrow_song_effect" Format="ia8" Width="32" Height="64" Offset="0x460"/>
+        <Array Name="sScarecrowsSongFrustumVtx" Count="22" Offset="0xC60">
+            <Vtx/>
+        </Array>
+        <DList Name="sOceff4Texture0DL" Offset="0xDC0"/>
+        <DList Name="sOceff4Texture1DL" Offset="0xE28"/>
+        <DList Name="sOceff4Texture2DL" Offset="0xE90"/>
```

Comment:
what about the `@bug` of the "dlist fallthrough" thing that's mentioned in the current _gfx.c
https://github.com/zeldaret/oot/blob/master/src/overlays/actors/ovl_Oceff_Wipe4/z_oceff_wipe4_gfx.c#L100-L126
`sOceff4Texture2DL` also has geometry ("model")